test/dynamo/test_functions.py

```diff
@@ -1794,6 +1794,10 @@ def call_triton(
         # to be in the metadata, so there might be false negatives
         self.assertTrue("aten.copy" not in codes[0])
         self.assertTrue("aten.clone" not in codes[0])
+        if dynamic and grad:
```

Comment:
Can you add a comment here: the import thing we're checking is that there are **only** two tensor outputs in the compiled graph? (Which is how we're testing that AOTAutograd is correctly preserving input mutations in the graph in all cases)